openedx/tests/settings.py

```diff
@@ -71,6 +73,7 @@
     'openedx.core.djangoapps.self_paced',
     'milestones',
     'celery_utils',
+    'lms.djangoapps.completion.apps.CompletionAppConfig',
```

Comment:
Nit: I guess there's already an `lms.djangoapps` app in here - but it hurts to add another one.

Reply:
Yeah, I think that's going to keep happening until we refactor to draw cleaner lines between the apps.